core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -67,7 +67,7 @@ oppia.factory('UrlInterpolationService', ['alertsService',
 
       // Parameter values can only contain alphanumerical characters, spaces,
       // hyphens or underscores.
-      var VALID_PARAMETER_VALUE_REGEX = /^(\w| |_|-)+$/;
+      var VALID_PARAMETER_VALUE_REGEX = /^(\w| |_|-|\[|\]|'|")+$/;
```

Comment:
It might be nice to rename this to VALID_URL_PARAMETER_VALUE_REGEX or maybe just URL_PARAM_VALUE_REGEX, since parameters have another connotation elsewhere.